modules/tribes/client/components/TribesHeader.component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import Board from '../../../core/client/components/Board';
```

Comment:
There a nice webpack alias to avoid all the `../../../` noise :)

Reply:
fixed in 76d84e9c3e396256b181a84f3b55836b3edcac1a